tests/test_integration/test_models/test_creator/test_model_creator.py

```diff
@@ -105,3 +105,65 @@ def test_session_summary_details(self, mock_return):
         actual = model.app
 
         assert_frame_equal(actual, expected)
+
+    @requests_mock.Mocker()
+    def test_keyword_search(self, mock_return):
+        """ Keyword search should return the proper DataFrames."""
```

Comment:
Please add a reference to the issue number in the docstring

Reply:
Fixed